packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -679,4 +679,45 @@ MeetingUtil.startInternalStats = (meeting) => {
   return meeting.internalStats;
 };
 
+MeetingUtil.handleAudioLogging = (audioTrack) => {
+  const LOG_HEADER = 'MeetingUtil#handleAudioLogging -->';
+
+  LoggerProxy.logger.log(LOG_HEADER);
+
+  if (audioTrack) {
+    const settings = audioTrack.getSettings();
+    const {deviceId} = settings;
+
+    LoggerProxy.logger.log(LOG_HEADER, deviceId);
+    LoggerProxy.logger.log(LOG_HEADER, 'settings', JSON.stringify(settings));
+  }
+};
+
+MeetingUtil.handleVideoLogging = (videoTrack) => {
+  const LOG_HEADER = 'MeetingUtil#handleVideoLogging -->';
+
+  LoggerProxy.logger.log(LOG_HEADER);
+
+  if (videoTrack) {
+    const settings = videoTrack.getSettings();
+    const {deviceId} = settings;
+
+    LoggerProxy.logger.log(LOG_HEADER, `deviceId = ${deviceId}`);
+    LoggerProxy.logger.log(LOG_HEADER, 'settings', JSON.stringify(settings));
+  }
+};
+
+MeetingUtil.handleDeviceLogging = (devices) => {
+  const LOG_HEADER = 'MeetingUtil#handleDeviceLogging -->';
+
+  LoggerProxy.logger.log(LOG_HEADER);
+
+  if (devices && devices.length) {
```

Comment:
can this be called without devices?  can you just default devices to be an array and not have to do this check